apps/03TicTacToe.js

```diff
@@ -24,23 +24,62 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) ||
+  (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) ||
+  (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn)){
+    return true;
+  }
 }
 
 function verticalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn) ||
+  (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn) ||
+  (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn)){
+    return true;
+  }
 }
 
 function diagonalWin() {
-  // Your code here
+  if ((board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn) ||
+  (board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn)){
+    return true;
+  }
 }
 
 function checkForWin() {
-  // Your code here
+  if ((horizontalWin() || verticalWin() || diagonalWin()) === true ){
```

Comment:
If you pass function(s) in an `if` statement, it will automatically evaluate the 'truthiness' of whatever that function returns.

ex: `if(returnTrue() || returnFalse()) { return "One of these functions returned true!" }`